games/r.yaml

```diff
@@ -118,6 +118,24 @@
   type: remake
   updated: 2015-07-29
 
+- name: Rednukem
+  type: remake
+  originals:
+  - Redneck Rampage
+  repo: https://github.com/nukeykt/NBlood
+  url: https://forums.duke4.net/topic/10241-rednukem-redneck-rampage-port/
+  development: sporadic
```

Comment:
sporatic? It was updated yesterday.

Reply:
The repository is active, but it contains 3 ports.  Rednukem hasn't been updated since 2018, so I chose sporadic rather than Active.  I checked latest releases first to determine correct status

Reply:
I don't know what you checked, but this commit is fixing stuff in Rednukem:
https://github.com/nukeykt/NBlood/commit/db89be2afe4ba595b24540ceb44ec9ecf558eb3b

Reply:
Fair enough.